service.projcontrol/resources/settings.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
Please convert the settings file to the new settings format.  The old settings format stills works, but for forward compatibility, we'd like everyone to start using the new format now.

https://kodi.wiki/view/Add-on_settings_conversion#Converting_addon_settings

One of the Team Kodi members wrote an addon that will convert other addon settings that will probably get you 95% of the way there (or might do it all).

https://gitlab.com/ronie/script.settings-convert